optashift-employee-rostering-server/src/test/java/org/optaplanner/openshift/employeerostering/server/skill/AbstractClientArquillianTest.java

```diff
@@ -0,0 +1,46 @@
+package org.optaplanner.openshift.employeerostering.server.skill;
+
+import java.io.File;
+import java.io.IOException;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.archive.importer.MavenImporter;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public abstract class AbstractClientArquillianTest {
+
+    private static final String POM_DIRECTORY_NAME = "optashift-employee-rostering-webapp";
+
+    @Deployment(testable = false)
+    public static WebArchive createDeployment() {
+        File file = findPomFile();
+        return ShrinkWrap.create(MavenImporter.class)
+                .loadPomFromFile(file)
+                .importBuildOutput()
+                .as(WebArchive.class);
+    }
+
+    private static File findPomFile() {
+        File file = new File("pom.xml");
```

Comment:
this means it's using the working directory,which should be the module directory as per maven standards. optaplanner-webexamples does this assumption too and there it works fine.